### ISSUE TITLE
Some people cant connect!?

### ISSUE THREAD

**Issue by CalvinDKHD** [AJS-development/Ogar-unlimited#998]
So this is the server for me and most others - https://gyazo.com/23d47f04016404718a646c427414e867
This is the server for some other people - https://gyazo.com/50b30abff69a4638c85905ce6129d52f

I only have 1 Multiverse server named Main. Which originally comes with Ogar UL.
But some people see Main and Main2 which i put in the HTML instead of FFA, TEAMS and Experimental. 
This is the HTML - https://gyazo.com/ff5194de32bce6315b2bceffc48a1257

**Comment by LegitSoulja** [comment]
Muticraft?

**Comment by LegitSoulja** [comment]
And you don't edit the HTML, it does it by itself... If you do that then you have to make your OWN system which connects to your multiverse servers. The first 2 pictures was the same btw..

**Comment by CalvinDKHD** [comment]
Yes, even before i edited the HTML this was happening. I will change it back to the original.

**Comment by CalvinDKHD** [comment]
Same problem / https://gyazo.com/29eda56c1dc74bdbc5bc933961648286

**Comment by LegitSoulja** [comment]
make sure you have a updated client and server

**Comment by CalvinDKHD** [comment]
ok let me try again.

**Comment by CalvinDKHD** [comment]
Send me the newest client please.

**Comment by CalvinDKHD** [comment]
OK i fixed it, so the skin thing still isnt working. Do you know why?

**Comment by Andrews54757** [comment]
it doesnt work like that. You dont edit the html. If you make a multiverse server, it automatically changes the client. Also, multiverse is cross-port, not cross-ip, so redirecting to different ports are okay, but ips, are not
